tests/recorder/recorder.go

```diff
@@ -0,0 +1,233 @@
+package main
```

Comment:
FWIW, i sort of liked autoplay although I really like the connection between "scope" and watching the cli do its thing